### PR TITLE
[10.0] migrated to new API module web_action_conditionable

### PR DESCRIPTION
**Pull request by ovnicraft**

### CODE REVIEW

**Comment by ovnicraft** [comment]
anyone for review it ?

**Comment by moylop260** [comment]
@oscarolar @hugho-ad
Could you help to review it, please?

**Review by hugho-ad — APPROVED** [review]

File: web_action_conditionable/__manifest__.py [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 {
     "name": 'web_action_conditionable',
-    "version": "8.0.0.1.0",
+    "version": "10.0.0.1.0",
```
Comment (simahawk) [inline web_action_conditionable/__manifest__.py]: shouldn't be 10.0.1.0.0?

**Review by simahawk — CHANGES_REQUESTED** [review]
Beside the version question I'd update xml view declaration to use ``<odoo />`` instead of ``<openerp /><data />`` see https://github.com/ovnicraft/web/blob/b7848efb45c613c33506455a7544eaf9cc0f883f/web_action_conditionable/views/view.xml

**Comment by ovnicraft** [comment]
Done.
Ref for #435

**Review by Fenkiou — APPROVED** [review]
Tested and works as expected 👍

**Comment by ovnicraft** [comment]
Hi ! possible merge this ?

**Pull request merged**